### PR TITLE
Add links to module pairings

### PR DESCRIPTION
**Pull request by Carldkennedy**
Add note pointing to table of modules to load for each version of TensorFlow

### CODE REVIEW

**Comment by jkwmoore** [comment]
I think this might need to be a warning rather than a note, as well as being shifted to the top of the GPU install section. 

I think it would benefit people more if it is more prescriptive with the instructions about compatibility been versions and module files being *essential*.

File: bessemer/software/apps/tensorflow.rst [hunk 1]
```diff
@@ -97,7 +100,7 @@ you should specify a version number when running ``pip install`` i.e. ::
    pip install tensorflow-gpu==<version_number>
 
 .. note::
-   You can find which modules to load for various versions of TensorFlow :ref:`here<tensorflow_cudnn_compat_bess>`.
+   You can find which CUDA and cuDNN module to load for various versions of TensorFlow :ref:`here<tensorflow_cudnn_compat_bess>`.
```
Comment (jkwmoore) [inline bessemer/software/apps/tensorflow.rst:103]: Can we just move these into the warning?

Comment (Carldkennedy) [reply bessemer/software/apps/tensorflow.rst:103]: Could do. I though it was nice to have the note right after the part showing how to use other versions. Leading into selecting the correct pairing

Comment (jkwmoore) [reply bessemer/software/apps/tensorflow.rst:103]: I don't think it makes much sense to effectively start a call to action, then defer to lower down? Seems less clear / user friendly. Arguably, the first thing anyone should do before installing it is ensure the versions match so I wanted to get ahead of them trying it, then implying they should have done X first.

**Pull request merged**